_sections/block/asheville-connection-agent-partners.md

```diff
@@ -2,10 +2,10 @@
 type: block
 title: asheville-connection-agent-partners
 block-title: asheville-program-blocks
-link: /asheville-connection-agent-partners/
+link: /partners/
```

Comment:
Can we please also make a redirect when we rename pages, so that they don't 404 for any established links?

Reply:
Agreed but I don't believe I actually renamed any pages -- as requested I just switched that link to point from one existing page to another

Reply:
My mistake.

Reply:
np 👍